cmd/expand.go

```diff
@@ -70,13 +70,25 @@ func (c *ExpandCmd) Run(args []string) error {
 		if err != nil {
 			return fmt.Errorf("error reading from stdin: %v", err)
 		}
-	} else if len(args) == 1 {
-		src, err = ioutil.ReadFile(args[0])
-		if err != nil {
-			return fmt.Errorf("error reading file %q: %v", args[0], err)
-		}
 	} else {
-		return fmt.Errorf("expected exactly one argument, a path to a file to expand")
+		for argi, filepattern := range args {
+			files, err := filepath.Glob(filepattern)
+			if err != nil {
+				return fmt.Errorf("error bad pattern %q: %v", filepattern, err)
+			}
+			for filei, file := range files {
+				filesrc, err := ioutil.ReadFile(file)
+				if err != nil {
+					return fmt.Errorf("error reading file %q: %v", file, err)
+				}
+				if argi == 0 && filei == 0 {
+					src = filesrc
+				} else {
+					filesrc = append([]byte("\n---\n"), filesrc...)
```

Comment:
This looks fine; I would probably just append to a byte buffer regardless, but you still need special logic .  I think if people do `kexpand... *.yaml` the shell will expand it.  I think if they do `kexpand "*.yaml"` filepath.Glob will expand it.   My only concern here is whether we should warn (or even error?) on an empty glob match, but I think this is good